<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItem</class>
 <widget class="QWidget" name="UserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_namel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_feeling">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>2201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
